src/protagonist/API/Features/Customer/Converters/CustomerConverter.cs

```diff
@@ -23,6 +23,7 @@ public static DLCS.HydraModel.Customer ToHydra(this DLCS.Model.Customers.Custome
     
     /// <summary>
     /// Converts the EF model object to a simplified JObject with just @id and @type
+    /// (PK): Could this return dedicated type, e.g. a record or sth?
```

Comment:
is this to make the generated classes easier to understand? this was possibly written before `records`  existed, definitely possible to convert it to one, but would this just be pushing the conversion around as at some point it would need to be serialized anyway?

Reply:
Mostly so that the API description is well, describing the output, not "any JObject" :D